src/features/board/components/board-issue-drawer.tsx

```
@@ -913,30 +915,53 @@ const BlockingSection = ({
 
 		return (
 			<div className="space-y-1.5">
-				{blockedByIssues.map((blockingIssue) => (
+				{blockedByIssuesDetailed.map(({ issue: blockingIssue, resolutionSteps, reasoning }) => (
 					<div
-						className="flex items-center justify-between p-2 rounded-md border bg-card hover:bg-accent/50 transition-colors cursor-pointer"
+						className="p-2 rounded-md border bg-card hover:bg-accent/50 transition-colors cursor-pointer"
 						key={blockingIssue._id}
 						onClick={() => onClickIssue(blockingIssue)}
 					>
```

Biome (2.4.14)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)